### ISSUE TITLE
Remove binaries from repository

### ISSUE THREAD

**Issue by max107** [AlwaysGeeky/Vox#182]

**Comment by AlwaysGeeky** [comment]
The plan is to remove these at some point soon in the future, for the moment they are there as an artifact for people to test the game on windows and linux without having to build the source code.

**Comment by AlwaysGeeky** [comment]
They are not always fully up to date with the latest code though, just rebuilt semi-occasionally.

**Comment by ZafotheNinja** [comment]
Indeed, I don't see this as a problem, very helpful to have them there.
